ouroboros-consensus/test-util/Test/Util/SOP.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
You know that there's somebody who would want you to make this change in a separate commit 🙂

Reply:
Thankfully that someone is on holidays :grin: